src/store/actions/sign-up.js

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
+import { SIGNUP } from './types';
+
+export const signUp = user => ({
+  type: SIGNUP,
+  user,
+});
+
+export const signUpAPI = user => dispatch => axios({
+  url: 'http://localhost:8000/api/sign-up',
```

Comment:
Abstract API calls to a middleware. This avoids having to define the baseUrl (http://localhost:8000/api` for every different API call. You can also use the middleware to handle setting the access token in the header.